_conferences/mobile-at-scale-2023.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "Mobile @Scale"
+website: https://atscaleconference.com/events/mobile-scale-2023/
+location: Global
+online: true
+
+date_start: 2023-10-25
+date_end:   2023-10-25
+
+cfp:
+    start: 2023-07-27
+    end:   2023-08-18
```

Comment:
Where are the CFP start/end dates mentioned? I couldn't find any information about them

Reply:
https://www.facebook.com/atscaleevents/posts/pfbid0Kgn4iPQAVisuXJpEKDgRzT6qmwEEXsQe5JuDHgpk6EB2sUMDSA91UMK6S71BxCril